### PR TITLE
Broaden anglerfish run search

### PR DESCRIPTION
**Pull request by kedhammar**

### CODE REVIEW

**Comment by kedhammar** [comment]
related https://github.com/SciLifeLab/TACA/pull/409

File: scripts/parse_anglerfish_results.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def find_latest_flowcell_run(currentStep: Process) -> str:
 
 
 def find_latest_anglerfish_run(latest_flowcell_run_path: str) -> str:
-    anglerfish_query = f"{latest_flowcell_run_path}/*anglerfish_run*"
+    anglerfish_query = f"{latest_flowcell_run_path}/**/anglerfish_run*"
```
Comment (alneberg) [inline scripts/parse_anglerfish_results.py:38]: I belive the double asterisk is sort of a special feature of some shells, have you tested this on the LIMS server?

Comment (kedhammar) [reply scripts/parse_anglerfish_results.py:38]: Tried it now

```
In [6]: p="repos/**/scripts"

In [7]: from glob import glob

In [8]: pwd
Out[8]: '/opt/gls/clarity/users/glsai'

In [9]: glob(p)
Out[9]: ['repos/scilifelab_epps/scripts']
```

Seems to work :)

**Review by alneberg — APPROVED** [review]
Looks good!

**Pull request merged**